features/storage/TESTS/blockdevice/heap_block_device/main.cpp

```diff
@@ -24,7 +24,7 @@
 using namespace utest::v1;
 
 // TODO HACK, replace with available ram/heap property
-#if defined(TARGET_MTB_MTS_XDOT)
+#if defined(TARGET_MTB_MTS_XDOT) || defined(TARGET_LPC1114)
```

Comment:
I'd like us to remove target dependant code whether possible, by I assume this is a temporary workaround.

Reply:
I agree - we should remove target dependant code, especially because this will quickly become unmanagable as more Mbed 2 targets are moved to bare metal. 
@SeppoTakalo

Reply:
If there is no better build-time macros to check heap sizes, then is is best we can do.